api_app/visualizers_manager/visualizers/passive_dns/visualize_report.py

```diff
@@ -0,0 +1,66 @@
+from api_app.visualizers_manager.classes import Visualizer
+
+
+def visualize_report(report):
+    obj = {}
+    source_description = report.pop("source_description", "")
+    for [key, value] in report.items():
```

Comment:
I think we could change the names and make them more specific:
* visualize_report -> render_pdns_table
* report -> raw_pdns_data: List[...]
* obj -> visualizable_raw
* key -> column_name